static/downloads/home-sensor-data.csv

```diff
@@ -1,3 +1,4 @@
+#datatype measurement,tag,field,field,field,time

```

Comment:
It looks like this CSV file is used in a few Python client library examples. Just want to make sure the added annotation doesn't break anything.